doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1711,27 +1868,19 @@ NCSDK-21709: :ref:`peripheral_uart` sample does not work on nRF52810 and nRF5281
 
   **Workaround:** Enable the :kconfig:option:`CONFIG_BT_SETTINGS` Kconfig option in the project configuration file (:file:`prj_minimal.conf`).
 
-.. rst-class:: v2-4-2 v2-4-1 v2-4-0
-
-NCSDK-21590: :ref:`bluetooth_mesh_sensor_client` sample does not compile for nRF52832
-  Adding mesh shell support for the :ref:`bluetooth_mesh_sensor_client` sample increases the need for RAM.
-  The :ref:`bluetooth_mesh_sensor_client` sample cannot compile because of RAM shortage.
-
-  **Workaround:** Disable the mesh shell support in the :file:`prj.conf` file for the sensor client sample.
-
 .. rst-class:: v2-3-0 v2-2-0
 
 NCSDK-18112: :ref:`bluetooth_central_dfu_smp` sample cannot do discovery on the :ref:`smp_svr_sample`
   The :ref:`bluetooth_central_dfu_smp` sample cannot perform the GATT discovery on a DK with the :ref:`smp_svr_sample`.
 
   **Workaround:** Enable the legacy LLCP mechanism (:kconfig:option:`CONFIG_BT_LL_SW_LLCP_LEGACY`).
 
-.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0 v0-4-0 v0-3-0
+.. rst-class:: wontfix v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0 v0-4-0 v0-3-0
 
 NCSDK-19942: HID samples do not work with Android 13
   The :ref:`peripheral_hids_keyboard` and :ref:`peripheral_hids_mouse` samples enter a connection-disconnection loop when you try to connect to them from a device that is running Android 13.
 
```

Comment:
Can we generalize this known issue and indicate that this issue affects all NCS releases as this is an Android bug?

```suggestion
NCSDK-19942: HID samples do not work with Android 13
  All Bluetooth samples and applications that are set up for the HIDS use case and have the Bluetooth Privacy feature enabled (:kconfig:option:`CONFIG_BT_Privacy`) enter a connection-disconnection loop when you try to connect to them from a device that is running Android 13.
  
See the list of affected samples and applications:

* :ref:`peripheral_hids_keyboard`
* :ref:`peripheral_hids_mouse`
* :ref:`fast_pair_input_device`
* Fast Pair configurations of the :ref:`nrf_desktop` application
```

FYI: @grochu, @KAGA164, @alstrzebonski, @MarekPieta

Reply:
It is ok to me to have one common entry in Known issue about Android 13

Reply:
@grochu, @KAGA164, are you sure that the HIDS keyboard sample does not work with Android 13? The keyboard sample does enable the CONFIG_BT_PRIVACY option and should work correctly

Reply:
You are right. It should be only peripheral hids mouse sample